src/main/resources/api/openapi.yaml

```diff
@@ -1652,15 +1652,15 @@ paths:
           application/json:
             schema:
               type: string
-              example: '7yaASMijGEGTbttYHg1MrXnWB8EbzjJnFLSWvmNoHrXV'
+              example: '"7yaASMijGEGTbttYHg1MrXnWB8EbzjJnFLSWvmNoHrXV"'
       responses:
         '200':
           description: Base16-encoded 32 byte hash
           content:
             application/json:
               schema:
                 type: string
-                example: '7e1e79dd4936bdc7d09f4ba9212849136b589fba4bcf4263a0961a95b65d08cb16'
+                example: '"6ed54addddaf10fe8fcda330bd443a57914fbce38a9fa27248b07e361cc76a41"'
```

Comment:
@kushti Why the value changed while in other cases it is not?

Reply:
@aslesarenko if the string is inside an object, Swagger generates quotes automatically